DJSR labels added

smrutib opened the pull request:
Added DJS Racing's labels and documentation

smrutib committed:
DJSR labels added

ddavid commented:
Hi Smruti, I will probably go through the PR tomorrow in the evening. I will keep you posted.

smrutib commented:
Thanks a lot

On Fri, Mar 27, 2020, 5:10 AM David Dodel <notifications@github.com> wrote:

> Hi Smruti, I will probably go through the PR tomorrow in the evening. I
> will keep you posted.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ddavid/fsoco/pull/68#issuecomment-604741465>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALD4GOMF7JNRKABCPTUX3ZTRJPRX5ANCNFSM4LSK7L4Q>
> .
>

ddavid commented:
Hi there,
so unfortunately there are a few issues so far.
I've only gone through the first ~300 images because at image `pwd117-cockpit00001038.jpg` there's a wrong class index. Your `classes` file has three classes, so please check that all the indices are in the range from [0-2].
There are also a few misclassified cones, here are two examples:
![boxes_pwd117-cockpit00000777](https://user-images.githubusercontent.com/18621443/77808775-113afe80-708d-11ea-9fc8-ccbb5c84fd3c.jpg)

![boxes_pwd117-cockpit00000816](https://user-images.githubusercontent.com/18621443/77808791-1e57ed80-708d-11ea-9d5c-469dd91c7a1a.jpg)

Furthermore, I appreciate the `info.txt` on what cones were left out, but please be more specific.

smrutib committed:
DJSR labels and info fixed

smrutib commented:
Hi David,

I have made the following changes:
1. Updated info file per your suggestions
2. Fixed the wrong class index and checked other labels for the same
3. Fixed incorrectly classified cones

Thanks
Best Regards,
Smruti
On Sat, Mar 28, 2020 at 5:29 AM David Dodel <notifications@github.com>
wrote:

> Hi there,
> so unfortunately there are a few issues so far.
> I've only gone through the first ~300 images because at image
> pwd117-cockpit00001038.jpg there's a wrong class index. Your classes file
> has three classes, so please check that all the indices are in the range
> from [0-2].
> There are also a few misclassified cones, here are two examples:
> [image: boxes_pwd117-cockpit00000777]
> <https://user-images.githubusercontent.com/18621443/77808775-113afe80-708d-11ea-9fc8-ccbb5c84fd3c.jpg>
>
> [image: boxes_pwd117-cockpit00000816]
> <https://user-images.githubusercontent.com/18621443/77808791-1e57ed80-708d-11ea-9d5c-469dd91c7a1a.jpg>
>
> Furthermore, I appreciate the info.txt on what cones were left out, but
> please be more specific.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ddavid/fsoco/pull/68#issuecomment-605361564>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALD4GOLSOYXHLZWP4KPYU33RJU4VHANCNFSM4LSK7L4Q>
> .
>

ddavid commented:
Hi Smruti,
thank you for the summary of the changes made. I will let you know as soon as I have feedback on the current version of the labels.

ddavid commented:
Hi Smruti,

the issues from the first comment are fixed, thank you.

I'm afraid I wasn't specific enough with my feedback on the `info.txt` file, though. The purpose behind it is for other teams to be able to pre-process the files to better fit their algorithms.
What a `very small cone` is might vary from team to team, your file should be clear as to what that means for yours. As a rule of thumb, if you can't put the information in quantifiable terms somehow, it won't be of much use for the other teams.
An example of this would be the size of the bounding boxes in Pixels, e.g. labeling all cones with a height > 10 Pixels.
Another example would be to set a height on the image plane up to which you've labeled. This cutoff line would only make sense if all images were taken with the camera in the same pose, more or less, making the assumption of a fixed horizon possible.


Regarding the quality of the labels, I will send you a sample with good examples, the bounding boxes should be tighter around the cones.
